data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/adjacent_abilities_recursion_by_id.cfg

```diff
@@ -0,0 +1,62 @@
+#textdomain wesnoth-test
+
+#####
+# API(s) being tested: [affect_adjacent][filter]ability_id_active=
+##
+# Actions:
+# Alice and Bob are both of type Test Melee Quintain.
+# Give Alice's and Bob leadership_test_recursion ability.
+# leadership_test_recursion active if adjacent unit has leadership_test_recursion active
```

Comment:
Give Alice and Bob a leadership_test_recursion ability, which is active only if an adjacent unit also has it active.

Reply:
fixed